test/xdg_desktop_portal_test.dart

```diff
@@ -459,6 +517,10 @@ class MockPortalServer extends DBusClient {
   final sessions = <MockPortalSessionObject>[];
   final usedTokens = <String>{};
 
+  final String? id;
```

Comment:
Recommend naming these `userId`, `userName` etc so it's clear what these are used for.